documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc

```diff
@@ -79,21 +85,24 @@ Slots can also be named so that only wanted content is added there. This is done
 </div>
 ----
 
-Now an element that could be for instance: `<label slot="title">Main header</label>` would not be positioned in the
-slot with the default "No content given!" but into the `<h1><slot name="title">...`
+* A `<label slot="title">Main header</label>` element, for example, will now NOT be positioned in the default "No content given!" slot, but rather in `<h1><slot name="title">…`​
+
+You can also nest named slots inside the main slot.
+
+*Example*: Nested slot elements.
 
-Named slots can also be used as default by nesting them inside the main slot like:
 [source,html]
 ----
 <slot name="fullName">
     <slot name="firstName"></slot>, <slot name="lastName"></slot>
 </slot>
 ----
 
-This will make the slot show data for light dom `slot="firstName"` `slot="lastName"` elements if no `slot="fullName"` is available and when a `<element slot="fullName">` is
-added it will "override/replace" the `firstName, lastName` data.
+* This makes the slot show data for light DOM `<slot="firstName">` and `<slot="lastName">` elements, if no `<slot="fullName">` element is available. If you add the `<slot="fullName">` element, it overrides and replaces the `firstName and lastName` data.
+
+The default slot and any named slot can contain multiple elements.
 
-The "default" slot and named slot can contain multiple elements.
+*Example*: Defining multiple elements for the default slot and multiple named slots.
```

Comment:
I think that the original piece of info is quite important:

> The "default" slot and named slot can contain multiple elements.

This info is kinda lost in the new rewording.

Reply:
Yeah, somewhat. Though, I would suggest we go with the original author's rewording on this one, if it's okay with you?